external/loaders/loaders/batches/_batch.py

```diff
@@ -158,8 +165,14 @@ def batches_from_mapper(
 
     if timesteps is None:
         timesteps = list(data_mapping.keys())
-    shuffled_times = np.random.choice(timesteps, len(timesteps), replace=False).tolist()
-    batched_timesteps = list(partition_all(timesteps_per_batch, shuffled_times))
+
+    if shuffle_timesteps:
+        final_timesteps = np.random.choice(
+            timesteps, len(timesteps), replace=False
+        ).tolist()
+    else:
+        final_timesteps = timesteps
```

Comment:
We realized in slack that there's no scenario where we'd want to leave the timesteps un-shuffled, so this can be reverted to the previous version.

Reply:
I'm confused. Isn't `shuffle_timesteps=False` required to iterate through the data sequentially...which in combination with the LRUCache accounts for like 90% of the speedup in the PR?

Reply:
To get this merged let's say keep the shuffle_timesteps (default True) option in. If there isn't a use case for it, it just won't get used.